snapcraft/tests/integration/store/test_store_push_metadata.py

```diff
@@ -54,6 +54,9 @@ def test_with_login(self):
         # Push the snap
         snap_file_path = '{}_{}_{}.snap'.format(name, version, 'all')
         self.assertThat(os.path.join(snap_file_path), FileExists())
+        output = self.run_snapcraft(['push', snap_file_path])
```

Comment:
Thanks for looking into this! Can we add a test which looks like the original but fails as no snap has been pushed before?

Reply:
Yes!